### DISCUSSION TITLE
Please help me !

### DISCUSSION THREAD

**Discussion by takuyasasaki734** [NAIST-SE/AttractingContributors#87] [General]
Please tell me about your favorite book name!
I will contribute your repository if you need!
https://github.com/TakuyaSasaki734/AttractingContributors-Book.git

**Comment by iikyara** [comment]
Added Hyoka!
Please help me too!
https://github.com/iikyara/AttractingContributors-Gifts

**Reply by shota-in** [reply]
I added.
Please help my task too.
https://github.com/shota-in/AttractingContributors-Songs

**Comment by eito-sa** [comment]
I added Bam and Kero.

I'd be happy to help me.
https://github.com/eito-sa/AttractingContributors-singers/blob/main/README.md

**Comment by shota-in** [comment]
I added.
Please help my task too.
https://github.com/shota-in/AttractingContributors-Songs
